### PR TITLE
Add Drivet URL Shortener API

### PR DESCRIPTION
**Pull request by GhostSlayer**
- [x] My submission is formatted according to the guidelines in the [contributing guide](/CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x] The description does not have more than 100 characters
- [x] The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [x] I have searched the repository for any relevant issues or pull requests
- [x] Any category I am creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1634,6 +1634,7 @@ API | Description | Auth | HTTPS | CORS |
 | [ClickMeter](https://support.clickmeter.com/hc/en-us/categories/201474986) | Monitor, compare and optimize your marketing links | `apiKey` | Yes | Unknown |
 | [Clico](https://cli.com/swagger-ui/index.html?configUrl=/v3/api-docs/swagger-config) | URL shortener service | `apiKey` | Yes | Unknown |
 | [Cutt.ly](https://cutt.ly/api-documentation/cuttly-links-api) | URL shortener service | `apiKey` | Yes | Unknown |
+| [Drivet URL Shortener](https://wiki.drivet.xyz/e/en/url-shortener/add-links) | Shorten a long URL easily and fast | No | Yes | Unknown |
```
Comment (matheusfelipeog) [inline README.md:1637]: Fix the link, please

image

The correct link is: https://wiki.drivet.xyz/en/url-shortener/add-links

```suggestion
| [Drivet URL Shortener](https://wiki.drivet.xyz/en/url-shortener/add-links) | Shorten a long URL easily and fast | No | Yes | Unknown |
```

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
Thanks for your contribution @GhostSlayer <3

Can you resolve thiis review, please?

**Comment by GhostSlayer** [comment]
Oops, looks like i linked the editing page accidentally, should be fixed now? :D

File: README.md (hunk 1 above)
Comment (AugustasV) [inline README.md:1637]: @GhostSlayer Current link still not working. Change URL from current one to https://wiki.drivet.xyz/en/url-shortener/add-links

**Review by AugustasV — CHANGES_REQUESTED** [review]

**Comment by GhostSlayer** [comment]
I see, i'll correct it immediately when i have time, probably in 30 minutes.

ma 8. marrask. 2021 klo 10.58 Augustas ***@***.***> kirjoitti:

**Review by matheusfelipeog — APPROVED** [review]
Now it's Ok ;)

Thanks for the update @GhostSlayer

**Pull request merged**